Rename bindings to prevent overflow in Jest

acdibble opened the pull request:
This is a fix for https://github.com/node-ffi-napi/ref-napi/issues/16, an issue that appears (especially in tests) when `writePointer` turns into an infinitely recursive call and overflows the stack.

This PR renames the native binding to already have the underscore to prevent the loop from ever being created.

acdibble committed:
Rename native binding to prevent infinitely recursive call

addaleax commented:
If you do this, can you follow through for all native bindings that follow this pattern, since it’s not just `writePointer` that’s affected as I understand it?

Also, the commit message would ideally point out that this works around a bug in Jest :)

acdibble committed:
Rename all native bindings that get renamed in ref.js

acdibble commented:
Changed all of the native bindings that followed the same pattern. Updated the PR title to mention Jest, but you can feel free to change it to whatever you feel is best.

addaleax commented:
Thanks, I’ll merge this after the CI runs :)

acdibble commented:
Thanks, I appreciate the responsiveness!

addaleax commented:
Published in `v2.1.2` and `v3.0.1`, thanks for the PR!
